pkg/virt-controller/services/template_test.go

```diff
@@ -1985,6 +1985,123 @@ var _ = Describe("Template", func() {
 
 		})
 
+		Context("with vhostuser interface", func() {
+			var (
+				vmi v1.VirtualMachineInstance
+			)
+			BeforeEach(func() {
+				vmi = v1.VirtualMachineInstance{
+					ObjectMeta: metav1.ObjectMeta{
+						Name:      "testvmi",
+						Namespace: "default",
+						UID:       "1234",
+					},
+					Spec: v1.VirtualMachineInstanceSpec{
+						Domain: v1.DomainSpec{
+							Devices: v1.Devices{
+								Interfaces: []v1.Interface{
+									v1.Interface{
+										Name: "test1",
+										InterfaceBindingMethod: v1.InterfaceBindingMethod{
+											Vhostuser: &v1.InterfaceVhostuser{},
+										},
+									},
+								},
+							},
+						},
+						Networks: []v1.Network{
+							{Name: "test1",
+								NetworkSource: v1.NetworkSource{
+									Multus: &v1.MultusNetwork{NetworkName: "test1"},
+								},
+							},
+						},
+					},
+				}
+			})
+			It("should throw error when memory is nil", func() {
+				_, err := svc.RenderLaunchManifest(&vmi)
+				Expect(err).To(HaveOccurred())
```

Comment:
It might be a good idea to check that the error this function returns is the one we expected.

Reply:
The hugepage validation has been moved to webhook, so this test has been removed along with the corresponding check on template.